package.json

```diff
@@ -23,7 +23,9 @@
     "flow": "flow check",
     "types": "shx rm -rf flow-typed && flow-typed install",
     "lint": "remark . && eslint .",
-    "test": "nyc -i ./lib/babel-hook.js --instrument false --source-map false mocha --opts mocha.opts"
+    "test": "nyc -i ./lib/babel-hook.js --instrument false --source-map false mocha --opts mocha.opts",
+    "debugger": "node test/utils/debugger/dist/debug.js",
+    "debugger-build": "rollup -c test/utils/debugger/rollup.config.js"
```

Comment:
This is completely subjective and isn't a big deal so feel free to ignore this if you would like.

I'm thinking `build:debugger` may be a better name for the debugger build script. In a vacuum `debugger-build` and `build:debugger` seem pretty similar but imo, `npm run build:debugger` reads better.